hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -176,7 +176,9 @@ public KafkaOffsetGen(TypedProperties props) {
     props.keySet().stream().filter(prop -> {
       // In order to prevent printing unnecessary warn logs, here filter out the hoodie
       // configuration items before passing to kafkaParams
-      return !prop.toString().startsWith("hoodie.");
+      return !prop.toString().startsWith("hoodie.")
+              // We need to pass some properties to kafka client so that KafkaAvroSchemaDeserializer can use it
+              || prop.toString().startsWith("hoodie.deltastreamer.source.kafka.value.deserializer.");
```

Comment:
reuse the property name from the other class, without hardcoding?

Reply:
This expression must currently match two settings:
`hoodie.deltastreamer.source.kafka.value.deserializer.class`
and
`hoodie.deltastreamer.source.kafka.value.deserializer.schema`
So we ether list all the settings (may be more than two in the future) which is error-prone or we use the prefix here

Reply:
understood. my point was to pull these into static final string constants. So its easy to trace if more places start using them

Reply:
Should i create a PR that pulls `hoodie.deltastreamer.source.kafka.value.deserializer.` to a static final string constant inside KafkaOffsetGen?
That would be no problem

Reply:
Created https://github.com/apache/hudi/pull/3172 that addresses this